app/dashboard/settings/page.tsx

```
@@ -1,25 +1,1169 @@
+
+    // Handle case where data has 'items' directly at root level
+    if (data && Array.isArray(data.items)) {
+      return {
+        result: {
+          items: data.items.map(convertOrderItem).filter(Boolean)
+        }
+      };
+    }
+
+    // Handle case where data has 'data' property containing orders
+    if (data && data.data && Array.isArray(data.data)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)